api/lib/application/sessions/session-controller.js

```diff
@@ -403,6 +403,15 @@ const deleteJuryComment = async function (request, h) {
   return h.response().code(204);
 };
 
+const dismissLiveAlert = async function (request, h) {
+  const { id: sessionId } = request.params;
+  const { userId } = request.deserializedPayload;
```

Comment:
🎌 Comme c'est pour de la traçabilité je pense que l'on ne peut pas faire confiance au payload, je propose d'utiliser ce qui est fait dans les autres API

```suggestion
  const { userId } = request.auth.credentials;
```

Reply:
si tu fait le `auth.credentials`, tu récupères le userId du surveillant. Or on souhaite le userId du candidat, qu'on récupère dans le payload ici